### ISSUE TITLE
Remove paste(s) feature

### ISSUE THREAD

**Issue by petersem** [enchant97/hasty-paste#21]
**Is your feature request related to a problem? Please describe.**
There is no "simple" way to remove old posts

**Describe the solution you'd like**
Provide an admin feature to view all pastes, which offers a delete feature. 

**Describe alternatives you've considered**
- Opening every paste folder and file, then examining the paste to see if you need to keep it. 
- Having the option to `blanket-delete` all pastes older than a set time.

**Comment by enchant97** [comment]
Thanks for the suggestion a future update with a cleanup function is already planned, as well as an admin interface, through CLI and not on the web UI.

**Comment by enchant97** [comment]
@petersem I've been working on the CLI. Do these commands suite your requirements?

This command will delete all pastes older than 365 days.

```
cli cleanup -y --older-than 365
```

This command will delete all pastes older than 365 days, and any that are expired:

```
cli cleanup -y --expired --older-than 365
```

**Comment by enchant97** [comment]
Implemented in [V1.4.0](https://github.com/enchant97/hasty-paste/releases/tag/v1.4.0)
